Buildasaur.xcodeproj/project.pbxproj

```diff
@@ -84,6 +84,9 @@
 		3AD338B01AAE31D500ECD0F2 /* BuildTemplateViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3AD338AF1AAE31D500ECD0F2 /* BuildTemplateViewController.swift */; };
 		3AF090B81B1134AA0058567F /* BranchWatchingViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3AF090B71B1134AA0058567F /* BranchWatchingViewController.swift */; };
 		3AF1B1241AAC7CA500917EF3 /* StatusSyncerViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3AF1B1231AAC7CA500917EF3 /* StatusSyncerViewController.swift */; };
+		55A43BAC1B8CDE4300EED079 /* ProjectViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55A43BAB1B8CDE4300EED079 /* ProjectViewController.swift */; settings = {ASSET_TAGS = (); }; };
+		6604F68F1B8F4E4900E70BA2 /* XcodeServerSDK.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6604F68E1B8F4E4900E70BA2 /* XcodeServerSDK.framework */; };
+		6604F6901B8F4E4900E70BA2 /* XcodeServerSDK.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 6604F68E1B8F4E4900E70BA2 /* XcodeServerSDK.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
Try to clone it cleanly in a new folder and just run `pod install` - that should be all you need to get it working (also make sure to merge `gladiolus` from my original repo, maybe you're behind on the `Podfile` dependencies?)